editor/group_settings_editor.cpp

```diff
@@ -0,0 +1,498 @@
+/*************************************************************************/
+/*  group_settings_editor.cpp                                            */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2022 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2022 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#include "group_settings_editor.h"
+
+#include "core/config/project_settings.h"
+#include "editor/editor_scale.h"
+#include "editor/scene_tree_dock.h"
+#include "editor_node.h"
+#include "scene/resources/packed_scene.h"
+
+void GroupSettingsEditor::_notification(int p_what) {
+	switch (p_what) {
+		case NOTIFICATION_ENTER_TREE: {
+			update_tree();
+		} break;
+	}
+}
+
+HashMap<StringName, String> GroupSettingsEditor::get_groups() const {
+	return _groups_cache;
+}
+
+void GroupSettingsEditor::_item_edited() {
+	if (updating_group)
+		return;
```

Comment:
Could you help with this? What I need to do? This is my first pull request with reviewing :)
1) Press "Commit suggestion"
2) Press "Resolve conversation"
3) Fix this in my branch and make force push with changes

Reply:
Sure! I find option 3 to be easiest, because pressing "Commit suggestion" makes a new commit which you then have to squash with the other commit. If you have to do that for several suggestions, then it makes a lot of commits to squash. 

"Resolve conversation" is for when the change is made, its decided that the change doesn't need to be made, or the change doesn't apply anymore. So here you can press it after you commit the suggestion.